tensorrt_llm/tokenizer/tokenizer.py

```
@@ -420,33 +420,41 @@ def load_hf_tokenizer(model_dir: str,
-            truncation_side='left',
-            trust_remote_code=trust_remote_code,
-            use_fast=use_fast,
-            **kwargs)
-
+        tokenizer = TransformersTokenizer.from_pretrained(model_dir, **load_kwargs)
         if trust_remote_code:
             maybe_register_transformers_modules_by_value()
-
         return tokenizer
-
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)